### ISSUE TITLE
Withdraw Funds

### ISSUE THREAD

**Issue by RedJocker** [BratinaRok/android-bank-manager#32]
I don't understand the withdraw funds feature.
How can a user of the app get money from their mobile device?

**Comment by BratinaRok** [comment]
This app is meant for learning purposes, for students. So with add Funds option, it simulates as if you go to the bank and deposit money or like from ATM,  into the account, and with the option withdraw you can take money from the account out.

**Comment by BratinaRok** [comment]
Yes, I understand what you mean, it doesn't make sense. I will change it to transfer money.
